tests/docker/start.irods.ubuntu20.sh

```diff
@@ -22,6 +22,9 @@ sudo -H -u irods bash -c "iadmin moduser user1 password user1"
 sudo -H -u irods bash -c "iadmin mkuser alice rodsuser"
 sudo -H -u irods bash -c "iadmin moduser alice password apass"
 
+#### Create user1 and alice in iRODS ####
+sudo -H -u irods bash -c "iadmin mkresc newResc unixfilesystem `hostname`:/tmp/newRescVault"
```

Comment:
Is this for debugging purposes or to improve testing in some way?

Reply:
this is to exercise the non-default resource name.

Reply:
Cool cool.